mne/time_frequency/csd.py

```diff
@@ -809,7 +809,7 @@ def csd_multitaper(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
 def csd_array_multitaper(X, sfreq, t0=0, fmin=0, fmax=np.inf, tmin=None,
                          tmax=None, ch_names=None, n_fft=None, bandwidth=None,
                          adaptive=False, low_bias=True, projs=None,
-                         n_jobs=None, max_iter=150, *, verbose=None):
+                         n_jobs=None, max_iter=250, *, verbose=None):
```

Comment:
This is a behavior change with no deprecation, so it either needs a dep cycle or we can consider it a bugfix and add a `Bugs` `latest.inc` entry

Reply:
ah good point.  Hard to justifiably call it a bug though, unless you say "we should have been using periodic (not symmetric) tapers, and when we correct that bug, we also need to bump up `max_iter`".

Reply:
I can agree with that quote :smile: